### DISCUSSION TITLE
Change slider judgements from "hit" to "miss"

### DISCUSSION THREAD

**Discussion by Givikap120** [ppy/osu#29981] [Ideas]
Currently there are 2 new slider judgements - Sliderend Hits and Slidertick Hits
But those judgements alone are useless without context of total amount of Sliderends and Sliderticks
This isn't much of an issue on result screen, but it's making in-game judgements count display essentially useless

I propose changing those 2 judgements to "Sliderend Misses" and "Sliderbreaks"
Second one is already unofficial judgement even in stable

**Comment by bdach** [comment]
What are you even proposing here? I read the op twice and still don't get it.

**Reply by Givikap120** [reply]
![image](https://github.com/user-attachments/assets/52aacc9a-5285-49f1-96c7-832e7b146444)

this thing
change "slider tick" and "slider end" to "sliderbreak" and "sliderend miss"
because the current judgement gives 0 information about the play

**Reply by shiumano** [reply]
Why not make it an addition instead of a change?
Normal judgment shows both hits and misses.

**Reply by Givikap120** [reply]
> Why not make it an addition instead of a change? Normal judgment shows both hits and misses.

Whatever makes UI convenient
This includes score screen UI because I don't want to do math every time when I want to get number of sliderbreaks

**Reply by peppy** [reply]
Seems like a valid proposal.
